src/test/java/org/opensearch/security/test/helper/rest/RestHelper.java

```diff
@@ -121,6 +127,26 @@ public String executeSimpleRequest(final String request) throws Exception {
 		}
 	}
 
+	public HttpResponse[] executeMultipleAsyncPutRequest(final int numOfRequests, final String request, String body) throws Exception {
+		try (CloseableHttpAsyncClient client = getHTTPAsyncClient(numOfRequests)) {
+			client.start();
+			List<Future<org.apache.http.HttpResponse>> futures = new ArrayList<>();
+			HttpResponse[] responses = new HttpResponse[numOfRequests];
+			HttpPut uriRequest = new HttpPut(getHttpServerUri() + request);
```

Comment:
@vrozov it can not find the endpoint if i use `getHttpServerUri() + "/" + request`, it will only works with `getHttpServerUri() +  request`. It seems to be the problem of `CloseableHttpAsyncClient` in the newer version of the library.

Reply:
Is it necessary to use async http client? Consider:
```
	public HttpResponse[] executeMultipleAsyncPutRequest(final int numOfRequests, final String request, String body) throws Exception {
		final ExecutorService executorService = Executors.newFixedThreadPool(numOfRequests);
		Future<HttpResponse>[] futures = new Future[numOfRequests];
		for (int i = 0; i < numOfRequests; i++) {
			futures[i] = executorService.submit(() -> executePutRequest(request, body, new Header[0]));
		}
		return Arrays.stream(futures)
				.map(HttpResponse::from)
				.toArray(s -> new HttpResponse[s]);
	}
```

Reply:
@vrozov this will lead to compile error since there is no `from` method in `HttpResponse`.

Reply:
@andy840314 I assumed that you will implement it. Something like:
```
		private static HttpResponse from(Future<HttpResponse> future) {
			try {
				return future.get();
			} catch (Exception e) {
                                 throw new RuntimeException(e);
			}
		}
```

Reply:
I forgot to shutdown `executorService` in the example code.

Reply:
Done.